test/e2e/kb/telemetry_test.go

```diff
@@ -57,6 +59,9 @@ func TestTelemetry(t *testing.T) {
 					if err != nil {
 						return err
 					}
+					if stats == nil || len(stats) == 0 {
```

Comment:
I think the second condition is technically enough (also works on pointers)

Reply:
Yes!